#159295598 Feedback

e0works opened the pull request:
### What does this PR do?
Implement LFA feedback
### Description of task to be completed ?
Implement the following:

Move the login modal to center of page
Click outside login modal should close login modal
Remove Register column on how it works section
Add icons to navigation menu items of dashboard page
Align add new entry button on navigation menu to fit in like other menu items
Remove user details from settings page
Align user details vertically on profile page
Align user activity vertically on profile page
Add background image behind profile picture on profile page
Change background color of list entry page
Move index.html from the root directory to UI directory

### How should this be tested
Navigate to the hosted templates at the link below
https://victor-ugwueze.github.io/MyDiary/UI

### What are the relevant pivotal tracker stories?
#159295598
### Any background context you wish to provide?
None

### Screenshorts ?
screenshots
![login_modal](https://user-images.githubusercontent.com/24890605/43187752-3cd4ac42-8fea-11e8-9963-35dec88f71dc.png)
![dashboard_menu_with_icons](https://user-images.githubusercontent.com/24890605/43187884-a730f4b0-8fea-11e8-8331-9731064f54cb.png)
![setting_page](https://user-images.githubusercontent.com/24890605/43187972-ec3d74de-8fea-11e8-93d7-0f29ac7f7acc.png)
![profile_page](https://user-images.githubusercontent.com/24890605/43188030-13af2fda-8feb-11e8-97e2-ef8e9072e51c.png)

e0works committed:
Fix login modal

Login modal postion fix at center, and click outside the modal closes the modal

e0works committed:
Remove register column

Register column of how it works section is not neccarry, hence removed

e0works committed:
Add icons to naviagtion bar

Added icons to menu items on navigation bar, and aligned add new entry menu item properly,

e0works committed:
Remove user details from settings page

The user details on settings page has been removed, because it's duplication of functinality
as it is also contained on profile page

e0works committed:
Refactore profile page

Made the following changes
  * Aligned user details and added inputs fields for updating user details
  * Aligned user activity on a box
  * Added backgroung image to behind the profile picture, previously it was background color

e0works committed:
Change background of list entry page and add notification icon

e0works committed:
Refacotor project structure

Moved index.html into UI directory, this is done to maintain a clean project structure

Celoka commented on UI/Resources/js/dashboard.js:
```diff
@@ -12,27 +12,27 @@ const rightSection = document.querySelector('#right-section');
+  button.addEventListener('click', function(){
     const modalToshow = document.querySelector('#'+this.dataset.target);
     SelectElement(modalToshow,null,'show');
   });
```
Online 43 of dashboard.js, don't you think it would be nice if you made that function an ES 6 function. Use arrow functions. Apply these wherever you have instances of these type of function.

e0works replied:
Thank you. I have changed all occurrences of callback function to ES6 Arrow function.

e0works committed:
Refactor all javascript code to be in ES6

Change callback function to use ES6 arrow function
Change toggleLogin object to LoginForm class

[chore-#159300201] this finish implementation of feedback
